crates/cargo-test-support/src/lib.rs

```diff
@@ -706,6 +808,14 @@ impl Execs {
     /// its output.
     ///
     /// See [`compare`] for supported patterns.
+    ///
+    /// <div class="warning">
```

Comment:
Cool! I didn't know rustdoc has special CSS classes until today.

Reply:
I think they actually have something newer but its not documented.  I'll ping them about it.